docs/api/autogluon.tabular.rst

```diff
@@ -0,0 +1,12 @@
+autogluon.tabular
+=================
+
+
+.. currentmodule:: autogluon.tabular
+
+.. autosummary::
+    :nosignatures:
+    :toctree: _autogen
+
+    TabularDataset
+    TabularPredictor
```

Comment:
This is awesome. However, the URL is very strange:

http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-2924/a5fb2a6/api/_autogen/autogluon.tabular.TabularPredictor.html#autogluon.tabular.TabularPredictor.clone

Compared to install URL:

http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-2924/a5fb2a6/install.html

Lets just focus on the suffix:

api/_autogen/autogluon.tabular.TabularPredictor.html#autogluon.tabular.TabularPredictor.clone

install.html

## Oddity 1

`api/_autogen` seems a bit odd.

Here is the URL for pandas `Series.to_frame`:

https://pandas.pydata.org/docs/reference/api/pandas.Series.to_frame.html

They don't have any `_autogen` in the path

## Oddity 2

`autogluon.tabular.TabularPredictor.html#autogluon.tabular.TabularPredictor.clone`

This is quite verbose.

Why not:

`autogluon.tabular.TabularPredictor.html#clone`

Alternatively, can we get a separate page for every method like is done with Pandas?

https://pandas.pydata.org/docs/reference/api/pandas.Series.to_frame.html

This looks really nice and is super easy to navigate.

The URL for a standalone page could be `auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.clone.html`